Classes/Domain/Model/Dto/EmConfiguration.php

```diff
@@ -126,4 +141,19 @@ public function getConfigurationArray(): array
     {
         return $this->configurationArray;
     }
+
+    public function getAuthorizeMode(): string
+    {
+        return $this->authorizeMode;
+    }
+
+    public function getRefreshToken(): string
+    {
+        return $this->refreshToken;
+    }
+
+    public function getExpires(): int
+    {
+        return (int)$this->expires;
```

Comment:
This integer cast should be obsolete?!

Reply:
Ne eigentlich nicht, da es ein input Feld ist. Wenn das Feld nicht gesetzt, dann ist es null oder leeren string.

Reply:
Default value is set by protected $expires = 0; and should not be a string or null